AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java

```diff
@@ -198,13 +206,68 @@ public void onBindViewHolder(ViewHolder holder, int position) {
+        }
+
         // Set the card counts and their colors
-        holder.deckNew.setText(String.valueOf(node.newCount));
-        holder.deckNew.setTextColor((node.newCount == 0) ? mZeroCountColor : mNewCountColor);
-        holder.deckLearn.setText(String.valueOf(node.lrnCount));
-        holder.deckLearn.setTextColor((node.lrnCount == 0) ? mZeroCountColor : mLearnCountColor);
-        holder.deckRev.setText(String.valueOf(node.revCount));
-        holder.deckRev.setTextColor((node.revCount == 0) ? mZeroCountColor : mReviewCountColor);
+        if (showRemainingCards) {
+            holder.countsLayout.setVisibility(View.VISIBLE);
+            holder.deckNew.setVisibility(View.VISIBLE);
+            holder.deckLearn.setVisibility(View.VISIBLE);
+            holder.deckRev.setVisibility(View.VISIBLE);
+            holder.deckNew.setText(String.valueOf(node.newCount));
+            holder.deckNew.setTextColor((node.newCount == 0) ? mZeroCountColor : mNewCountColor);
+            holder.deckLearn.setText(String.valueOf(node.lrnCount));
+            holder.deckLearn
+                .setTextColor((node.lrnCount == 0) ? mZeroCountColor : mLearnCountColor);
+            holder.deckRev.setText(String.valueOf(node.revCount));
+            holder.deckRev.setTextColor((node.revCount == 0) ? mZeroCountColor : mReviewCountColor);
+        } else {
+            // ViewHolder objects are (potentially) reused, so it's important to
+            // always be explicity about the state that you want. It's not enough to just have
+            // the above "if" put the views in the proper state, we need this
+            // else as well
+            holder.countsLayout.setVisibility(View.INVISIBLE);
+            holder.deckNew.setVisibility(View.INVISIBLE);
+            holder.deckLearn.setVisibility(View.INVISIBLE);
+            holder.deckRev.setVisibility(View.INVISIBLE);
+        }
+
+        // Show/hide the minimum interval information
+        if (showMinimumInterval) {
+            holder.minIvlLayout.setVisibility(View.VISIBLE);
+            holder.minIvl.setVisibility(View.VISIBLE);
+            int doneReviewingIvl = 14; // if minIvl is 14 days, you are g2g
+            if (node.minIvl > doneReviewingIvl) {
+                holder.minIvl.setText(HtmlCompat.fromHtml("\uD83D\uDC4D", HtmlCompat.FROM_HTML_MODE_LEGACY)); // thumbs up
+            } else {
+                holder.minIvl.setText(String.format("%dd", node.minIvl));
+            }
+            // also show any cards in learning stage
+            if (node.lrnCount > 0) {
+                holder.countsLayout.setVisibility(View.VISIBLE);
+                holder.deckLearn.setVisibility(View.VISIBLE);
+                holder.deckLearn.setText(String.format("(%d)", node.lrnCount));
+                holder.deckLearn
+                    .setTextColor((node.lrnCount == 0) ? mZeroCountColor : mLearnCountColor);
```

Comment:
General question: do we like the color red for these? To me red is associated with something bad, which a failed review really isn't (don't want to promote seeing failures as bad). Thoughts on going with another color?

Reply:
agree, red = bad. ultimately i think the absence of the 👍should communicate that you need to do work, so everything else can just be black?

fixed